spaces/models.py

```diff
@@ -0,0 +1,146 @@
+
+    class Meta:
+        db_table = 'sub_properties'
+
+
+class Option(models.Model):
+    space    = models.ForeignKey('spaces.Space', on_delete=models.CASCADE)
+    name     = models.CharField(max_length=45)
+    icon_url = models.URLField(max_length=2000)
+
+    class Mata:
+        db_table = 'options'
+
+
+class SpaceOption(models.Model):
+    space  = models.ForeignKey('spaces.Space', on_delete=models.CASCADE)
+    option = models.ForeignKey('spaces.Option', on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = 'space_options'
+
+
+class Location(models.Model):
+    space          = models.ForeignKey('spaces.Space', on_delete=models.CASCADE)
+    country        = models.ForeignKey('users.Country', on_delete=models.CASCADE)
+    region         = models.CharField(max_length=45)
+    city           = models.CharField(max_length=20)
+    address        = models.CharField(max_length=150)
+    address_detail = models.CharField(max_length=45)
+    letitude       = models.DecimalField(max_digits=10, decimal_places=6,null=True)
+    longitude      = models.DecimalField(max_digits=10, decimal_places=6,null=True)
+
+    class Meta:
+        db_table = 'locations'
+
+
+class Image(models.Model):
+    space     = models.ForeignKey('spaces.Space', on_delete=models.CASCADE)
+    image_url = models.URLField(max_length=2000)
+
+    class Meta:
+        db_table = 'images'
+
+
+class Detail(models.Model):
+    space      = models.ForeignKey('spaces.Space', on_delete=models.CASCADE)
+    max_people = models.IntegerField(default=0)
+    bathroom   = models.IntegerField(default=0)
+    
+    class Meta:
+        db_table = 'details'
+
+
+class Bedroom(models.Model):
+    space   = models.ForeignKey('spaces.Space', on_delete=models.CASCADE)
+    bedroom = models.IntegerField()
+
+    class Meta:
+        db_table = 'bedrooms'
+
+
+class BedroomBed(models.Model):
+    bedroom  = models.ForeignKey('spaces.Bedroom', on_delete=models.CASCADE)
+    bed_type = models.ForeignKey('spaces.BedType', on_delete=models.CASCADE)
+    quantity = models.IntegerField(default=0)
+
+    class Meta:
+        db_table = 'bedroom_beds'
+
+
+class BedType(models.Model):
+    name = models.CharField(max_length=20)
+
+    class Meta:
+        db_table = 'bed_types'
+           
+
+class Review(models.Model):
+    space         = models.ForeignKey('spaces.Space', on_delete=models.CASCADE)
+    user          = models.ForeignKey('users.User', on_delete=models.CASCADE)
+    pub_date      = models.DateField(auto_now=True)
+    content       = models.CharField(max_length=1000)
+    cleanliness   = models.IntegerField(default=0)
+    communication = models.IntegerField(default=0)
+    check_in      = models.IntegerField(default=0)
+    accuracy      = models.IntegerField(default=0)
+    location      = models.IntegerField(default=0)
+    value         = models.IntegerField(default=0)
+    image_url     = models.URLField(max_length=2000)
```

Comment:
아래와 같이 좀 더 필드의 목적을 잘 나타내도록 수정하면 좋을 것 같습니다.
```suggestion
    cleanliness_score   = models.IntegerField(default=0)
    communication_score = models.IntegerField(default=0)
    check_in_score      = models.IntegerField(default=0)
```

Reply:
네!!!